tests/network/hotplug.go

```diff
@@ -101,6 +101,21 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			waitForSingleHotPlugIfaceOnVMISpec(hotPluggedVMI)
 			hotPluggedVMI = verifyDynamicInterfaceChange(hotPluggedVMI, plugMethod)
 			Expect(libnet.InterfaceExists(hotPluggedVMI, vmIfaceName)).To(Succeed())
+
+			updatedVM, err := kubevirt.Client().VirtualMachine(hotPluggedVM.Namespace).Get(context.Background(), hotPluggedVM.Name, &metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			vmIfaceSpec := vmispec.LookupInterfaceByName(updatedVM.Spec.Template.Spec.Domain.Devices.Interfaces, ifaceName)
+			Expect(vmIfaceSpec).NotTo(BeNil(), "VM spec should contain the new interface")
+
+			Eventually(func(g Gomega) {
+				updatedVMI, err := kubevirt.Client().VirtualMachineInstance(hotPluggedVMI.Namespace).Get(context.Background(), hotPluggedVMI.Name, &metav1.GetOptions{})
+				g.Expect(err).NotTo(HaveOccurred())
```

Comment:
I think it should be an immediate failure and not cause a retry.

Reply:
This is the current repeated pattern applied all over the e2e code base.
If you want to change it, I suggest you start a discussion over this topic, unrelated to this PR main topic.

Reply:
IMO it's a mistake to not tolerate fetch errors on CI, but as you said this discussion out of this PR scope.

I changed to what you suggested, by omitting `g.` from the VMI fetching assertion the Eventually will fail immediately (on VM fetch error).